frb_codegen/src/utils/command_runner.rs

```diff
@@ -112,14 +105,10 @@ pub(crate) fn execute_command<'a>(
 
     let stdout = String::from_utf8_lossy(&result.stdout);
     if result.status.success() {
-        debug!(
-            "command={:?} stdout={} stderr={}",
-            cmd,
-            stdout,
-            String::from_utf8_lossy(&result.stderr)
-        );
```

Comment:
👀  seems we should not remove logs, o/w hard to debug.

but for the cargo expand, we can remove logs for *it* (eg add a flag)

Reply:
It is not a problem for cargo expand, it was just a lot of output. I changed it to only output in the case of an error, but we could change that back.

Reply:
I agree. Maybe we can add a function parameter to enable/disable this logging?

Reply:
i think it's fine to just always activate. If you don't want the output you could change the logging level.